bloom-housing/bloom
# Fix/infinite loop

## Issue

It's not covered by any issue.

Addresses # (issue)

## Description

This PR solves 2 problems:

1)
A language from the `choose-language` step is not saved by a conductor. I also refactored autofill to do not reset context (language is selecting in the first step and autofill is a secondary step - so we need to know what language the user selected)

2)
url: http://localhost:3000/applications/start/choose-language without `listingId` parameter occurs infinity loop with GET /listings call to the API.

## Type of change


## How Can This Be Tested/Reviewed?

1) Log in
2) Choose language different than EN (it was default previously so we want to test different case)
3) Submit application
4) Open public app in a new card (to reset context)
5) Log in
6) Post application with different language
7) See on the partners if an application includes selected language


## Checklist:
